java8-appengine/analytics/pom.xml

```diff
@@ -61,6 +61,14 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>${appengine.sdk.version}</version>
       </plugin>
+      <plugin>
```

Comment:
This might be a devEx issues where IDEs are not up to date with their Maven bundles...(IJ and/or NetBeans, not sure about Eclipse). Or why I did not get an error about maven version?